be/functions.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+
+
+function getAllRecords($db)
+{
+    $query = $db->prepare("SELECT `band`, `song`, `year`, `genre`, `link`, `image` FROM `record_collection`;");
+    $query->execute();
+    $result = $query ->fetchAll();
+    return $result;
+};
+
+
+
```

Comment:
add a doc block to describe what your function is doing- applies to all functions